protostar/commands/test/cheatcodes/reflect/reflector.py

```diff
@@ -0,0 +1,94 @@
+from copy import deepcopy
+
+from starkware.cairo.lang.vm.vm_consts import (
+    is_simple_type,
+    VmConstsReference,
+    VmConsts,
+)
+from starkware.cairo.lang.vm.relocatable import RelocatableValue
+from starkware.cairo.lang.compiler.ast.cairo_types import (
+    TypeStruct,
+    TypePointer,
+    TypeFelt,
+)
+
+from starkware.cairo.lang.compiler.identifier_definition import StructDefinition
+from protostar.commands.test.test_environment_exceptions import CheatcodeException
+
+from protostar.commands.test.cheatcodes.reflect.misc import (
+    ReflectInputType,
+    to_cairo_naming,
+    ReflectReturnType,
+)
+
+from protostar.commands.test.cheatcodes.reflect.get_from_vm import (
+    get_value_from_vm,
+)
+from protostar.commands.test.cheatcodes.reflect.build_output import (
+    generate_value_tree,
+)
+
+
+class Reflector:
+    def __init__(self, ids: VmConsts):
+        self._ids = ids
+        self._value: ReflectInputType = None
+
+    # We need to access Cairo's underscore variables
+    # pylint: disable=W0212
+    def __getattr__(self, name: str) -> "Reflector":
+        if not self._value:
```

Comment:
Why do we need `self._value` can't we just make it function local variable?

Reply:
I guess it doesn't matter too much.

Reply:
So better not to indroduce unused stuff